eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
+		"react/boolean-prop-naming": "error",
+		"react/button-has-type": "error",
+		"react/default-props-match-prop-types": "error",
+		"react/jsx-boolean-value": ["error", "never"],
+		"react/jsx-child-element-spacing": "error",
+		"react/jsx-closing-bracket-location": "error",
+		"react/jsx-closing-tag-location": "error",
+		"react/jsx-curly-brace-presence": "error",
+		"react/jsx-curly-spacing": ["error",  {
```

Comment:
i prefer without space

Reply:
Personally I don't care with spaces or not, I just don't want them mixed

Reply:
I prefer with :P

Reply:
why here and why not in function(argument), `my template ${ x }` ? call( x ) ?

Reply:
Because we are doing it for objects in regular JS:
https://github.com/Marfeel/MarfeelCodingStandards/blob/master/eslint/base/.eslintrc#L129

Reply:
but for me it's more like a parenthesis and a template literal than an object

Reply:
@marcmrf Yep... I can agree on it. ;)